src/legacy/ui/public/vislib/lib/chart_grid.js

```diff
@@ -24,7 +24,7 @@ export function VislibGridProvider() {
 
   const defaults = {
     style: {
-      color: '#eee'
+      color: '#69707d33',
```

Comment:
https://caniuse.com/#feat=css-rrggbbaa

Don't think this will work for IE

Reply:
So it actually converts it to proper `rgba` when the library applies it to the path:

![image](https://user-images.githubusercontent.com/549577/52733921-f6452200-2f91-11e9-8a26-2fe52401fd6a.png)

Reply:
But it looks like the TSVB ones do not..

Reply:
Changed them all to rgba. I think I thought it wasn't working because it was before I realized that the color was being saved with the visualization so refreshing wasn't affecting anything.